WcaOnRails/app/views/users/edit.html.erb

```diff
@@ -159,3 +159,17 @@
     <%= f.submit 'Save', class: "btn btn-primary" %>
   <% end %>
 </div>
+
+<script>
+$('.date_picker.form-control').on('dp.error', function(e){
```

Comment:
I'm happy to see you went with something less obstrusive than an `alert` =)

Do we only want this on the users edit page, not everywhere?

I haven't checked out exactly what this does, but did you consider using bootstrap's validation states? I feel like that might be all you need here. See <http://getbootstrap.com/css/#forms-control-validation>